### ISSUE TITLE
LensFlare shaders errors 

### ISSUE THREAD

**Issue by stefaniapedrazzi** [cyberbotics/webots#2528]
When opening the `highway.wbt` sample simulation (or any other world with LensFlare) I get these errors:
```
stefi@libeccio:~/webots$ ./webots projects/vehicles/worlds/highway.wbt 
An error occured during shader compilation/linking!
Vertex shader path: /home/stefi/webots/resources/wren/shaders/lens_flare.vert
Fragment shader path: /home/stefi/webots/resources/wren/shaders/lens_flare.frag
webots-bin: PostProcessingEffect.cpp:108: void wren::PostProcessingEffect::Pass::apply(): Assertion `locationTexture >= 0' failed.
./webots: line 88: 27958 Aborted                 (core dumped) "$webots_home/bin/webots-bin" "$@"
```

It works fine on the master branch.

**Comment by omichel** [comment]
I am getting a different warning on Windows. In the Webots console I get:
```
ERROR: Shader compilation failed! Filename: lens_flare.vert
0(6) : error C7614: GLSL ES doesn't allow use of reserved word noperspective
```
Which gives me some hints... I will investigate...

**Comment by stefaniapedrazzi** [comment]
Ok, thank you. I wanted to test #2281 directly on the develop branch but this is preventing loading the world.

**Comment by omichel** [comment]
Simply removing `noperspective ` from `webots/resources/wren/shaders/lens_flare.vert` and `webots/resources/wren/shaders/lens_flare.frag` seems to fix the problem (although the lens flare doesn't seem to work).

**Comment by omichel** [comment]
It seems that fixing this problem is not trivial because OpenGL ES doesn't have access to this `noperspective` functionality.
This would require that we pass an additional parameter to the shader and make changes to the shader.
However, the lens flare is anyhow not working properly (see #939).
I would suggest to disable it for now (by simply removing this noperspective keyword) and fix #939 later on.

**Comment by omichel** [comment]
The compilation failure was fixed in #2529.
The lens flare effect (which was anyhow [buggy](#939) is now disabled.
